blog/en/blog/2024/04/30/monthly-report.md

```diff
@@ -0,0 +1,37 @@
+---
+title: "Monthly Report (April 01 - April 30)"
+keywords: ["Apache APISIX", "API Gateway", "Monthly Report", "Contributor"]
+description: Our monthly Apache APISIX community report generates insights into the project's monthly developments. The reports provide a pathway into the Apache APISIX community, ensuring that you stay well-informed and actively involved.
+tags: [Community]
+image: https://static.apiseven.com/uploads/2024/04/30/XvSXCOav_monthly-report-apr-en.png
+---
+
+> We have recently made some additions and improvements to specific features within Apache APISIX. For detailed information, please read the monthly report.
```

Comment:
why's this different from

![image](https://github.com/apache/apisix-website/assets/39619599/eca109a0-5332-4f60-a296-760f206e2b96)